packages/fxa-settings/src/components/Settings/FlowRecoveryKeyHint/en.ftl

```diff
@@ -0,0 +1,12 @@
+## Fourth view in the Recovery-Key-Add UX redesign flow
+
+# This will be the title of the back button so that users know where that button will take them
+flow-recovery-key-hint-back-button-title = Back to settings
+# The header of the fourth view in the Recovery Key Add flow
+flow-recovery-key-hint-header = Help yourself find your key later
+# In the fourth and final of the Recovery Key Add flow, this message tells users why they would want to save a hint for themselves as to where they saved their recovery key, and what could happen if they lose the key.
+flow-recovery-key-hint-message = Remember where you saved your key by creating a hint. If you forget your password and can't find your key, you might not be able to get your data back.
```

Comment:
nit, not a smart quote here for "can't"

Reply:
This will also get flagged in l10n linters, so please do use `’`